src/index.js

```diff
@@ -49,10 +48,10 @@ class WebSockets {
     }
 
     return multiaddrs.filter((ma) => {
-      if (includes(ma.protoNames(), 'ipfs')) {
-        ma = ma.decapsulate('ipfs')
-      }
-      return mafmt.WebSockets.matches(ma) || mafmt.WebSocketsSecure.matches(ma)
+      return mafmt.WebSockets.matches(ma) ||
+        mafmt.WebSocketsSecure.matches(ma) ||
+        mafmt.WebSocketsIPFS.matches(ma) ||
+        mafmt.WebSocketsSecureIPFS.matches(ma)
```

Comment:
I wasn't sure when I coded this if it was the best way of doing it. The main idea behind leaving the non IPFS version was fear of breaking something else. I think in this case it should be safe to just do:

```
 mafmt.WebSocketsIPFS.matches(ma) ||
 mafmt.WebSocketsSecureIPFS.matches(ma)
```